src/Algorithms/BackwardEuler.jl

```diff
@@ -0,0 +1,53 @@
+"""
+$(TYPEDEF)
+
+Backward Euler's integration scheme with given step-size.
+
+### Fields
+
+- `Δt` -- step-size
+
+"""
+struct BackwardEuler{N} <: AbstractSolver
+    Δt::N
+end
+
+BackwardEuler(; Δt::T) where N = BackwardEuler(Δt)
+
+function _init(alg::BackwardEuler, M, C, K)
+    Δt = alg.Δt
+    K̂ = K * Δt + C
+    return K̂
+end
+
+function _solve(alg::BackwardEuler{N},
+                ivp::InitialValueProblem{<:SecondOrderAffineContinuousSystem{N}, XT},
```

Comment:
here we could dispatch on `Ex'(t) = Ax(t) + f(t)`, which is the purpose of https://github.com/JuliaReach/MathematicalSystems.jl/issues/230

Reply:
... but we can do it in a later step, in the meantime assuming `M  =  0`

Reply:
ok